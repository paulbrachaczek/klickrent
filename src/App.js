import React, { useState } from "react";
import Header from './components/header/header.component';
import Button from './components/button/button.component';
import Modal from './components/modal/modal.component';
import './App.css';

const App = () => {
  const [modalShow, setModalShow] = useState(false);
  const closeModal = () => {
    setModalShow(false);
  }
  
  return (
    <div className="App">
      <Header />
      <main className="o-main">
        <Button onClick={() => setModalShow(true)}>Start inquiry</Button>
        {modalShow ? <Modal closeModal={closeModal}/> : null}
      </main>
    </div>
  );
}

export default App;
