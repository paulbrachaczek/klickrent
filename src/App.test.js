import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";
import React from 'react';
import App from "./App";

it("renders correctly", () => {
    const AppComponent = shallow(<App />);
    expect(toJson(AppComponent)).toMatchSnapshot();
});
  