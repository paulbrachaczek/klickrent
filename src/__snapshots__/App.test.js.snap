// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <Header />
  <main
    className="o-main"
  >
    <Button
      onClick={[Function]}
    >
      Start inquiry
    </Button>
  </main>
</div>
`;
